assets/scripts/app/focus.js

```diff
@@ -44,18 +42,8 @@ export function onWindowFocus () {
   saveSettingsLocally()
 }
 
-function onWindowBlur () {
-  if (abortEverything) {
-    return
-  }
-
-  hideAllMenus()
-}
-
 function onVisibilityChange () {
-  if (document[system.visibilityState] === 'hidden') {
-    onWindowBlur()
```

Comment:
Whatever been in `onWindowBlur` is not necessary to be triggered when `system.visibilityChange` occurs and visibility state will be `hidden`?